apisix/admin/standalone.lua

```diff
@@ -151,9 +179,21 @@ local function get(ctx)
     local accept = core.request.header(nil, "accept") or "application/json"
     local want_yaml_resp = core.string.has_prefix(accept, "application/yaml")
 
-    local _, ver, config = config_yaml._get_config()
+    local config, err = get_config()
+    if not config then
+        if err ~= "not found" then
+            core.log.error("failed to get config from shared dict: ", err)
+            return core.response.exit(500, {
+                error_msg = "failed to get config from shared dict: " .. err
+            })
+        end
+        config = {}
+        local created_objs = config_yaml.fetch_all_created_obj()
+        for _, obj in pairs(created_objs) do
+            config[obj.conf_version_key] = obj.conf_version
+        end
```

Comment:
This branch indicates that the shared dict is not configured, so just return an empty object? There is no need to populate these version fields.

Reply:
It is used to view the currently available resources, as well as their corresponding confw_version keys and initial confw_version.
The purpose is for ease of use, and if you find it meaningless, I can delete it.

Reply:
I thought about it, and I think that when the shared dict is not found, it should return 500 with an error message indicating that this shared dict needs to be configured.

Reply:
Should the initial state be 500?
 I think the behavior of this interface itself should have no internal errors.
 Its purpose is to return the current configuration information.
 If not, it is empty.
@bzp2010  What's your opinion?

Reply:
Do we need to indicate any special meaning through the 500 response code of this interface? If it's just an empty object, I guess the client can understand that the server doesn't have any configurations to do the diff correctly, which doesn't seem to be a sufficient basis to support the default return of 500 and an error message?

@nic-6443 @AlinsRan 🤔

Reply:
So I think HTTP 200 + `{}` response would be sufficient.

Reply:
Sorry, I misunderstood. I thought the returned `not found` error was due to missing shared dict, but it is actually because no configuration has been submitted yet. It should not return 500; I agree with using 200.